vaultclient/src/test/java/com/scality/vaultclient/dto/CreateAccountResponseJsonUnmarshallerTest.java

```diff
@@ -0,0 +1,83 @@
+package com.scality.vaultclient.dto;
+
+import com.amazonaws.http.HttpResponse;
+import com.amazonaws.transform.JsonUnmarshallerContext;
+import com.google.gson.Gson;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class CreateAccountResponseJsonUnmarshallerTest {
+
+    private CreateAccountResponseJsonUnmarshaller createAccountResponseJsonUnmarshallerUnderTest;
+
+    @BeforeEach
+    void setUp() {
+        createAccountResponseJsonUnmarshallerUnderTest = CreateAccountResponseJsonUnmarshaller.getInstance();
+    }
+
+    @Test
+    void testUnmarshall() throws Exception {
+        // Setup
+        final CreateAccountResponseDTO expectedResult = new CreateAccountResponseDTO();
+        final Account account = new Account();
+        final AccountData data = new AccountData();
+        data.setArn("arn");
+        data.setName("name");
+        data.setEmailAddress("emailAddress");
+        data.setId("id");
+        data.setQuotaMax(0);
```

Comment:
qq: why are we setting this to 0?

Reply:
Actually this can be any value :)
Changing to 100